README.md

```diff
@@ -1,15 +1,245 @@
+  * Please use the github repository [SomaticShortV_makePON-nf](https://github.com/Sydney-Informatics-Hub/Somatic-shortV-makePON-nf) to create the PoN. 
+<br>
+
+(2) **Call the somatic short variants for a Tumor-Normal pair** :
+<br>The two main steps involved in calling  Somatic Short Variants are - 
+  * Generate a large set of candidate somatic variants.
+  * Filter the candidate somatic variants into a more confident set of somatic variant calls.
+
+- Calling candidate somatic short variants involves the use of the tool [Mutect2](https://gatk.broadinstitute.org/hc/en-us/articles/360035531132) which calls both SNVs and indels simultaneously by generating a local assembly of haplotypes in an active region, de-novo. When Mutect2 sees a region with somatic variations, it dis-regards the existing mapping information completely and re-assembles the reads in that region in order to generate candidate variant haplotypes.
+
+* This is followed by calculating contamination using the tools [GetPileupSummaries](https://gatk.broadinstitute.org/hc/en-us/articles/9570416554907-GetPileupSummaries) and [CalculateContamination](https://gatk.broadinstitute.org/hc/en-us/articles/9570322332315-CalculateContamination)
+* The next step is to learn the parameters of a model for orientation bias using the tool [LearnReadOrientationModel](https://gatk.broadinstitute.org/hc/en-us/articles/360051305331-LearnReadOrientationModel)
+* The [FilterMutectCalls](https://gatk.broadinstitute.org/hc/en-us/articles/360036856831-FilterMutectCalls) tool then applies filters to the raw output of Mutect2. 
+
+
+(3) **Annotating the variants** 
+<br> An external genomic variant annotations and functional effect prediction tool [SnpEff](http://pcingola.github.io/SnpEff/) is used for annotating the filtered variants (such as amino-acid changes etc). Please refer to the above link for SnpEff details.
+
+
+## Diagram
+
+<p align="center"> 
+<img src="./images/Somatic_variant_calling_SIH_modified.png" width="60%">
+</p> 
+
 
 ## Workflow description 
 
 ## User guide 
 
-## Component tools 
+To run this pipeline, you will need to prepare your input files, reference data, and clone this repository. Before proceeding, ensure Nextflow is installed on the system you're working on. To install Nextflow, see these [instructions](https://www.nextflow.io/docs/latest/getstarted.html#installation). 
+
+### 1. Prepare inputs
+
+To run this pipeline you will need the following inputs: 
+
+* Paired Tumor-Normal paired BAM files
+* Corresponding BAM index files  
+* Input sample sheet 
+
+This pipeline processes paired BAM files and is capable of processing multiple samples in parallel. BAM files are expected to be coordinate sorted and indexed (see [Fastq-to-BAM](https://github.com/Sydney-Informatics-Hub/Fastq-to-BAM) for an example of a best practice workflow that can generate these files).  
```

Comment:
Might also want to link to nf-core/sarek

Reply:
Added  link to nf-core/sarek

Reply:
Where?